src/types/base/locales.ts

```
@@ -1,4 +1,146 @@
+    addToCard: string
+  }
+  skuheader: {
+    skuId: string
+  }
+  addresslist: {
+    addAddress: string
+  }
+  comment: {
+    complaintsText: string
+    // eslint-disable-next-line @typescript-eslint/ban-types
+    additionalReview: Function
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)